app/data/Owners.scala

```diff
@@ -27,46 +27,55 @@ trait Owners {
 
 object Owners extends Owners {
 
-  override def default: Owner = Owner("dig.dev.tooling")
+  override def default: Owner = Owner("devx.sec.ops")
 
   override def stacks: Set[(String, SSA)] = Set(
-    "security.dev" -> SSA("security"),
+    "devx.sec.ops" -> SSA("security"),
+    "devx.sec.ops" -> SSA("deploy"),
+    "dotcom.platform" -> SSA(stack = "abacus"),
+    "dotcom.platform" -> SSA("domains"),
     "dotcom.platform" -> SSA("frontend"),
     "dotcom.platform" -> SSA("frontend-elk"),
     "identitydev" -> SSA(stack = "discussion"),
-    "dotcom.platform" -> SSA(stack = "abacus"),
     "identitydev" -> SSA(stack = "identity"),
-    "dig.dev.tooling" -> SSA("deploy"),
-    "dig.dev.tooling" -> SSA("domains"),
+    "digitalcms.dev" -> SSA("crosswords"),
     "digitalcms.dev" -> SSA("flexible"),
     "digitalcms.dev" -> SSA("flexible-secondary"),
     "digitalcms.dev" -> SSA("workflow"),
     "digitalcms.dev" -> SSA("cms-fronts"),
     "digitalcms.dev" -> SSA("elk-new"),
+    "digitalcms.dev" -> SSA("media-service"),
+    "digitalcms.dev" -> SSA("grid-elasticsearch"),
+    "digitalcms.dev" -> SSA("composer"),
+    "digital.investigations" -> SSA("pfi"),
     "digital.investigations" -> SSA("pfi-giant"),
     "digital.investigations" -> SSA("pfi-playground"),
     "digital.investigations" -> SSA("pfi-structured"),
-    "thegrid.dev" -> SSA("media-service"),
```

Comment:
`thegrid.dev` also includes Central Production, who probably don't want alerts about AWS infrastructure, so we now use `digitalcms.dev` instead